frontend/src/metabase/lib/urls/questions.ts

```diff
@@ -77,7 +77,11 @@ export function question(
   }
 
   if (mode === "notebook") {
-    path = `${path}/notebook`;
+    if (card.type === "model" || card.type === "metric") {
```

Comment:
I'm not sure if putting this condition here is a good idea.
It makes it ambiguous what `mode === "notebook"` means (it won't be possible to navigate to `/model/:id/notebook` via this function anymore). I don't have a counterexample where we wouldn't want this to happen though.

Still, I think it'd be better to introduce a new `mode` to make it unambiguous.